README.md

```diff
@@ -1,11 +1,25 @@
 # RAPIDS [devcontainers](https://containers.dev/)
 
-This repository contains features and workflows for building development containers to support local dev and CI for NVIDIA [RAPIDS](https://github.com/rapidsai), [CCCL](https://github.com/nvidia/cccl), and [Legate](https://github.com/nv-legate).
+This repository contains features and workflows for building development
+containers to support local dev and CI for NVIDIA
+[RAPIDS](https://github.com/rapidsai), [CCCL](https://github.com/nvidia/cccl),
+and [Legate](https://github.com/nv-legate).
+[Devcontainers](https://containers.dev/) are an open standard for specifying the
+creation and execution of Docker containers for developing a codebase. It's like
+using a docker image to develop, but there's some extra configuration and
+installation that can be done. It also provides some alternative ways of
+composing functionality and configuration that can augment Docker's
+capabilities.
 
-We've chosen to use a monorepo, but it is similar in spirit to the official [devcontainers/features](https://github.com/devcontainers/features) and [devcontainers/images](https://github.com/devcontainers/images) repositories.
+In addition to scripts that set up the devcontainer environment for things like GitHub auth, this repo
+contains reusable scripts to install software in arbitrary containers, aiding in composition and code sharing.
+A "feature" in VSCode terms refers to these installation scripts. The script for each feature runs when
+creating the devcontainer.
```

Comment:
Removing line breaks in paragraphs.

```suggestion
In addition to scripts that set up the devcontainer environment for things like GitHub auth, this repo contains reusable scripts to install software in arbitrary containers, aiding in composition and code sharing. A "feature" in VSCode terms refers to these installation scripts. The script for each feature runs when creating the devcontainer.
```

Reply:
I don't really like this verbiage.

This repo doesn't contain reusable scripts to install into arbitrary containers, it contains features to compose into a container via the [@devcontainers/cli](https://github.com/devcontainers/cli). That may sound like a nitpick, but it is functionally important, because a user building a Docker container can't just `ADD <feature_url>; RUN install.sh` to execute the logic.

The devcontainer CLI is a Dockerfile generator, and the code it generates for each layer includes envvars and sets image metadata for fields in the `devcontainer-feature.json`, without which the `install.sh` won't execute successfully.

The phrase, "a feature in VSCode terms," doesn't make sense to me., The devcontainer and features specification is unrelated to VSCode. VSCode is a tool that implements the spec.

Again, maybe we should just copy descriptions from (or link to) the relevant [devcontainer reference](https://containers.dev/implementors/json_reference/) and [features reference](https://containers.dev/implementors/features/) pages?